app/src/main/java/org/jellyfin/androidtv/ui/playback/PlaybackController.java

```diff
@@ -86,6 +87,7 @@
     private VideoOptions mCurrentOptions;
     private int mDefaultSubIndex = -1;
     private int mDefaultAudioIndex = -1;
+    private double mRequestedPlaybackSpeed = -1.0;
```

Comment:
Why not 1.0 by default?

Reply:
When I was writing this my thought process was to fail very obviously if for some reason we get the playback controller going without the playback action configuring a speed.
This should help us notice any bugs where the playback action somehow isn't created before the controller, which would also break carrying a users speed forwards

Reply:
Sounds like a good idea

Reply:
I'm more on the side of breaking gracefully and creating useful debug output, and not impacting the user experience

Reply:
So coming from a C/C++ background is there some way of detecting if were in Debug mode:

`mRequestedPlaybackSpeed = DEBUG ? -1.0 : 1.0`
That way it "just works" for users but should be painfully obvious for devs?

Reply:
What I was thinking is just set it to 1.0 so you don't "debug by letting stuff break". Then you could have a timber.d() debug log somewhere that would catch it at the beginning of the playback session lifecycle. You could probably put it in play() or the onPrepared listener